code/ui/blocks/src/blocks/mdx.tsx

```diff
@@ -177,17 +179,14 @@ const HeaderWithOcticonAnchor: FC<HeaderWithOcticonAnchorProps> = ({
           }
         }}
       >
-        <svg
-          viewBox="0 0 16 16"
-          version="1.1"
-          width="16"
-          height="16"
-          aria-hidden="true"
-          fill="currentColor"
-        >
+        <svg viewBox="0 0 14 14" width="14px" height="14px">
```

Comment:
why not import from our icon set instead?

Reply:
I'll give it a go.